test/es6/letconst_global_shadow_builtins.baseline

```diff
@@ -1,4 +1,6 @@
 [object Math]
+ReferenceError: Use before declaration

```

Comment:
Is Use before decl what we want for these cases? Or would Redeclaration be more intuitive?